src/document_formatting_edit_provider/dbtDocumentFormattingEditProvider.ts

```diff
@@ -52,11 +68,11 @@ export class DbtDocumentFormattingEditProvider
       .filter((s) => s !== "");
 
     const sqlFmtArgs = [
+      "-",
```

Comment:
Why we need this dash?

Reply:
This is an argument to input stdin to sqlfmt. ([ref](https://docs.sqlfmt.com/getting-started/using-sqlfmt#:~:text=sqlfmt%20can%20also%20format%20code%20passed%20through%20standard%20input%20(stdin)%20by%20passing%20%2D%20as%20the%20files%20argument.%20The%20formatted%20code%20will%20be%20output%20to%20stdout%20(all%20other%20output%20from%20sqlfmt%20is%20routed%20to%20stderr)%3A)). Instead of passing document.uri.fsPath to sqlfmt, putting this here enables stdin input to sqlfmt and the .complete function passes document's content to sqlfmt through stdin.